src/EditorFeatures/Core/Implementation/CallHierarchy/Finders/OverridingMemberFinder.cs

```diff
@@ -24,7 +24,7 @@ public override string DisplayName
         {
             get
             {
-                return EditorFeaturesResources.Overrides;
+                return EditorFeaturesResources.Overrides_;
```

Comment:
Ending with an _ , intended? What does this convention mean? or was it meant to be `Overrides_{0}` or something?